args/args.go

```diff
@@ -48,9 +47,9 @@ func (d *DefaultArgumentList) HasInventory() bool {
 // HTTPClientArgumentList are meant to be used as flags from a custom integrations. With this you could
 // send this arguments from the command line.
 type HTTPClientArgumentList struct {
-	HTTPCaBundleFile string        `default: "" help: "Name of the certificate file"`
-	HTTPCaBundleDir  string        `default: "" help: "Path where the certificate exists"`
-	HTTPTimeout      time.Duration `default:30 help: "Client http timeout in seconds"`
+	HTTPCaBundleFile string        `default:"" help:"Name of the certificate file"`
+	HTTPCaBundleDir  string        `default:"" help:"Path where the certificate exists"`
+	HTTPTimeout      time.Duration `default:"30" help:"Client http timeout in seconds"`
```

Comment:
shouldn't this be `"30s"`?

Reply:
just saw that the sdk cannot parse time.Duration arguments, changing it to int

Reply:
It should be able to parse `time.Duration` from a string into something like seconds.